modules/administration/pages/ssl-certs.adoc

```diff
@@ -20,5 +20,52 @@ Using a third party CA makes certificates harder to spoof, and will provide addi
 By default, {productname} uses a self-signed certificate.
 If you have certificates signed by a third party CA, you can import them to your {productname} installation.
 
+
+
+== SSL Certificates in {productname}
+// I'm adding this as a separate section for now, to make this easier, but I think we can probably merge these two sections together. --LKB 2020-06-29
+
+Every {productname} Server that uses SSL requires an SSL server certificate.
+Your server certificate is stored using the ``SERVER_CERT`` environment variable.
```

Comment:
We need to take care that the context of this chapter is correct.
Otherwise nobody understand about what environment variables we talk here.

AFAIK this is only used for the setup at new installations. For the process of replacing certs, we have commandline options which are named similar but not exactly the same.

Reply:
Heh, of course :roll_eyes:

Reply:
Maybe we should name both. First would be to express that the variable contain a path to the file.

```suggestion
Provide the path to the server certificate either in the environment variable ``SERVER_CERT`` during setup or with ``--from-server-cert`` when using rhn-ssl-tool directly.
```

Similar for `CA_CERT` and `--from-ca-cert` .